content/blog/getting-started-with-pulumi-audit-logs/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Getting Started with Pulumi Audit Logs"
```

Comment:
Maybe we can make this pop a bit more for someone not already deep into Pulumi:

# Auditing Your Organization's Infrastructure as Code Activity

?

Reply:
Yes, I like it. 👍